Assets/Scripts/Entity/AbstractEntity.cs

```diff
@@ -46,5 +52,20 @@ public void FixedUpdate()
             else
                 _Animator.SetBool("Falling", false);
         }
+        public void OnTriggerEnter2D(Collider2D other) 
+        {
+        //Check the provided Collider2D parameter other to see if it is tagged "PickUp", if it is...
+        if (other.gameObject.CompareTag("Collectible"))
```

Comment:
Add a tab here, so code would be aligned properly

Reply:
Files changed